src/Configuration.h

```diff
@@ -82,6 +84,7 @@ class Configuration
   byte EE_NUM_EVS = 0;
   byte EE_PRODUCED_EVENTS = 0;
   byte EE_BYTES_PER_EVENT; // Value calculated in begin()
+  unsigned int EE_VLCB_END; // Value calculated in begin()
```

Comment:
I am trying to come up with a better name. "VLCB" is not describing this well. How about EE_CONFIG_END or EE_MODULE_END or EE_USER_START ?

Reply:
Or EE_FREE_BASE to match the getter function?